pkg/virtctl/vmexport/vmexport.go

```diff
@@ -664,6 +841,26 @@ func handleDownloadFlags() error {
 	return nil
 }
 
+// handleManifestFlags ensures that only compatible flag combinations are used with 'manifest'
+func handleManifestFlags() error {
+	if pvc != "" {
+		return fmt.Errorf(ErrSupportedExportType)
+	}
+
+	if outputFile != "" {
+		return fmt.Errorf(ErrIncompatibleFlag, OUTPUT_FLAG, MANIFEST)
```

Comment:
Did that, now download supports both `--output` and not, if not specified it will write to stdout. This means if you don't specify --output, it will start streaming to stdout. So you can pipe to a file or another application.